internal/policy/container/has_required_labels.go

```diff
@@ -86,3 +67,9 @@ func (p *HasRequiredLabelsCheck) Help() check.HelpText {
 			"and validate that they do not violate Red Hat trademark.",
 	}
 }
+
+// getDataForValidate is a helper function to obtain the labels from an images configfile
+func getDataForValidate(image cranev1.Image) (map[string]string, error) {
```

Comment:
Might move this to a `labels.go` or something. Since it is shared, it should be easier to find.